enterprise_extensions/gp_kernels.py

```diff
@@ -31,6 +28,8 @@ def linear_interp_basis_dm(toas, freqs, dt=30*86400):
     return U * Dm[:, None], avetoas
 
 # linear interpolation basis in time with nu^-4 scaling
```

Comment:
These comments should be docstrings.  That will eliminate the need for extra whitespace

Reply:
Oh yeah. There were a lot of them in this file. I've put them all into the docstrings.